### PR TITLE
Fix GitHub Actions Workflow

### PR DESCRIPTION
**Pull request by pjbollinger**
Testing this locally with https://github.com/nektos/act, I was able to reproduce the issue and fix it with this change.
Now that the workflow has been committed to the repository, hopefully it will be easier to test and check.

Apologies for the additional PR, I should have looked to see how to test it locally before committing. 😅

### CODE REVIEW

File: .github/workflows/test-and-lint.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install dependencies
         run: npm install
       - name: Compile TypeScript
-        run: tsc
+        run: npx tsc
```
Comment (raucao) [inline .github/workflows/test-and-lint.yml:23]: The npm test script actually includes compiling TypeScript already, so I think this step is not necessary.

Comment (pjbollinger) [reply .github/workflows/test-and-lint.yml:23]: I agree 👍  I wasn't sure exactly why this separate `tsc` was called out in the first place, other than a sanity check, but I'll go ahead and remove this step.

Comment (raucao) [reply .github/workflows/test-and-lint.yml:23]: Yeah, just saw that it was already unnecessary on Travis. Just an oversight when we were working on the whole TS port I guess.

**Comment by raucao** [comment]
:tada:

**Pull request merged**